persistence.py

```
@@ -0,0 +1,132 @@
+
+        config_loaded = self._load_config()
+        state_loaded = self._load_state(combined_data)
+
+        if not config_loaded and isinstance(combined_data, dict):
+            cfg = combined_data.get("config")
+            if cfg:
+                try:
+                    self._config = AppConfig.from_dict(cfg)
+                    config_loaded = True
+                except Exception:
+                    pass
```

Ruff (0.13.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)